frontend/src/app/components/filters-form/filters-form.component.ts

```diff
@@ -1,24 +1,38 @@
-import { Component, inject, OnInit } from "@angular/core";
+import { Component, inject, OnInit ,EventEmitter, Output} from "@angular/core";
 import { TaskStatus } from "../../models/task-status";
 import { AppStore } from "../../store/app.store";
 import { AppState } from "../../models/app-state";
+import { FormControl } from "@angular/forms";
 
 @Component({
   selector: 'app-filters-form',
-  templateUrl: 'filters-form.component.html'
+  templateUrl: 'filters-form.component.html',
+  styleUrls: ['filters-form.component.scss']
 })
 export class FiltersFormComponent implements OnInit {
   protected TaskStatus = TaskStatus;
   protected appStore: AppStore = inject(AppStore);
   protected currentStatus: TaskStatus;
+  public formSearchString: FormControl = new FormControl('',);
```

Comment:
The `formSearchString` is not a form. It is a control. I suggest renaming to `searchStringControl`

Reply:
Done